[15.0][MIG] purchase_force_invoiced: Migration to 15.0

AnnaPForgeFlow opened the pull request:
@ForgeFlow
Standard migration to 15.0

JordiBForgeFlow committed:
[11.0][add][purchase_force_invoiced]

HviorForgeFlow committed:
Apply suggestions from code review

Co-Authored-By: Eric @ Elico Corp <eric.caudal@elico-corp.com>

oca-travis committed:
[UPD] Update purchase_force_invoiced.pot

OCA-git-bot committed:
[UPD] README.rst

MiquelRForgeFlow committed:
[MIG] purchase_force_invoiced: Migration to 12.0

oca-travis committed:
[UPD] Update purchase_force_invoiced.pot

OCA-git-bot committed:
[UPD] README.rst

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (4 of 4 strings)

Translation: purchase-workflow-12.0/purchase-workflow-12.0-purchase_force_invoiced
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-12-0/purchase-workflow-12-0-purchase_force_invoiced/zh_CN/

kongrattapong committed:
[IMP] purchase_force_invoiced: black, isort

kongrattapong committed:
[MIG] purchase_force_invoiced: Migration to 13.0

oca-travis committed:
[UPD] Update purchase_force_invoiced.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-13.0/purchase-workflow-13.0-purchase_force_invoiced
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_force_invoiced/

Reyes4711-S73 committed:
[14.0][MIG] purchase_force_invoiced: Migration to 14.0

oca-travis committed:
[UPD] Update purchase_force_invoiced.pot

OCA-git-bot committed:
[UPD] README.rst

AnnaPForgeFlow committed:
[IMP] purchase_force_invoiced: black, isort, prettier

AnnaPForgeFlow committed:
[MIG] purchase_force_invoiced: Migration to 15.0

MiquelRForgeFlow commented on purchase_force_invoiced/readme/CONTRIBUTORS.rst:
```diff
@@ -0,0 +1,2 @@
+* Jordi Ballester <jordi.ballester@eficent.com>
```
```suggestion
* Jordi Ballester <jordi.ballester@forgeflow.com>
```

MiquelRForgeFlow commented on purchase_force_invoiced/__manifest__.py:
```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
```
-> ForgeFlow

pedrobaeza committed:
[FIX] purchase_force_invoiced: Adapt BI report to the check

The purchase BI analysis is not taking into account the
"Force invoiced" check for showing their quantity to be billed, so it
gives contradictory information from the search filter and the pivot.

This commit adapts the SQL report for taking this into account.

pedrobaeza commented:
This is already migrated in #1485

sergio-teruel commented:
Can you please include #1531 ?
There is other PR openend https://github.com/OCA/purchase-workflow/pull/1485 but @Olageibol does not answer

MiquelRForgeFlow commented:
@sergio-teruel PR is ready. I would close the other old PR.

sergio-teruel commented:
Thanks @MiquelRForgeFlow

pedrobaeza commented:
/ocabot migration purchase_force_invoiced
/ocabot merge nobump
